Added the fonts required for the setup (#108)
elenapan/dotfiles: README.md

   - You will need to create the directory if it does not exist.
   - It does not matter that the actual font files (`.ttf`) are deep inside multiple directories. They will be detected as long as they can be accessed from `~/.fonts` or `~/.local/share/fonts`.

>    You can find the fonts required inside the fonts folder in misc

Reviewer: ```suggestion
   You can find the fonts required inside the `misc/fonts` folder of the repository.
```

Changed wording a bit.

   ```shell

>    cp -r ./misc/fonts/* ~/.fonts/  [ or to ~/.local/share/fonts/]

Reviewer: ```suggestion
   cp -r ./misc/fonts/* ~/.fonts/
   # Or to ~/.local/share/fonts
   cp -r ./misc/fonts/* ~/.local/share/fonts/
```
Small change here: I would prefer every command to be in its own line, so that it is easily copy-pastable.
